lib/index.js

```diff
@@ -1,44 +1,28 @@
-module.exports = {
-  extends: [
-    'eslint:recommended',
-  ],
-  parserOptions: {
+const globals = require("globals")
+const js = require("@eslint/js")
+const pluginNode = require("eslint-plugin-node")
+const pluginImport = require("eslint-plugin-import")
+const pluginPromise = require("eslint-plugin-promise")
```

Comment:
Does not yet support ESLint 9: https://github.com/eslint-community/eslint-plugin-promise/issues/449 We will get to it within the `eslint-community`

See also: https://github.com/neostandard/neostandard/issues/14